peerconnection_media_test.go

```diff
@@ -372,7 +372,6 @@ func TestPeerConnection_Media_Shutdown(t *testing.T) {
 	pcAnswer.OnICEConnectionStateChange(func(iceState ICEConnectionState) {
 		if iceState == ICEConnectionStateConnected {
 			go func() {
-				time.Sleep(3 * time.Second) // TODO PeerConnection.Close() doesn't block for all subsystems
```

Comment:
Unrelated, but noticed this when reading code! 

We shouldn't need this anymore. In `pion/transport` we sleep and check, so don't need this